app/MMB/src/scenes/explore/ExploreBookingMulticity.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import { TextIcon } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { TitledMenuGroup, MenuItem } from '@kiwicom/mobile-navigation';
+import idx from 'idx';
+
+import type { ExploreBookingMulticity as ExploreBookingMulticityType } from './__generated__/ExploreBookingMulticity.graphql';
+
+type Props = {|
+  +data: ExploreBookingMulticityType,
+|};
+
+const warnTodo = () => console.warn('TODO');
+
+const renderItem = (isAirport: boolean, title, description, leg) => (
+  <MenuItem
+    title={<Translation passThrough={title} />}
+    description={<Translation passThrough={description} />}
+    icon={isAirport ? <TextIcon code="a" /> : <TextIcon code="c" />}
+    isActive={false}
+    onPress={warnTodo}
+    key={getDepartureAirportCity(leg)}
+  />
+);
+
+const getDepartureAirportCity = leg =>
+  idx(leg, _ => _.departure.airport.city.name) || '';
+const getDepartureAirportCode = leg =>
+  idx(leg, _ => _.departure.airport.code) || '';
+const getDepartureAirportCountry = leg =>
+  idx(leg, _ => _.departure.airport.country.name);
+
+const ExploreBookingMulticity = (props: Props) => {
+  const trips = idx(props, _ => _.data.trips);
```

Comment:
if you do -> `const trips = idx(props, _ => _.data.trips) || [];`
then you can skip this check `if (trips && trips.length > 0)` and just do `trips.map`